### PR TITLE
fix(sql): improve sql help section with query variables

### PR DESCRIPTION
**Pull request by burgerni10**

### CODE REVIEW

File: src/south/SQL/SQL.schema.jsx [hunk 1]
```diff
@@ -98,30 +98,58 @@ schema.form = {
...
-          For example, if @EndTime and @StartTime is a two hours interval, and maxReadInterval is 3600s, the query will be split in two parts.
-          Two files will be created.
+          The query must be adapted to the selected driver (MSSQL, Oracle...).
+          The following variable can be used in the query:
+          <ul>
+            <li>
+              <b>@StartTime:</b>
+              {' '}
+              The time of the previous query retrieved from the &apos;Time column&apos; field.
+              For the firs query, it is initialized to the current date.
```
Comment (kukukk) [inline src/south/SQL/SQL.schema.jsx:108]: first

File: src/south/SQL/SQL.schema.jsx [hunk 2]
```diff
@@ -98,30 +98,58 @@ schema.form = {
...
           To prevent blocking if the SQL server is not available or the query is faulty it is possible to configure
-          separate connection timeout and request timeout.
+          separate connection timeout and request timeout (not available for sqlite).
+        </p>
+        <p>
+          To avoid a heavy query on a server, the query can be split into several sub-queries.
+          To do so, you can use the variables @StartTime and @EndTime.
           <br />
+          If you request data with a column named timestamp, the WHERE clause can look like this :
+          WHERE timestamp &gt; @StartTime and timestamp &lt; @StartTime.
```
Comment (kukukk) [inline src/south/SQL/SQL.schema.jsx:130]: timestamp &lt; @EndTime

File: src/south/SQL/SQL.schema.jsx [hunk 3]
```diff
@@ -98,30 +98,58 @@ schema.form = {
...
+        </p>
+        <p>
+          @StartTime is set to the last retrieved value (retrieved from the &apos;Time column&apos; field),
+          and @EndTime is set to @StartTime + maxReadInterval (if @StartTime + maxReadInterval &lt; current date).
+          @StartTime is then incremented at each sub-query until it @EndTime reaches the current date.
```
Comment (kukukk) [inline src/south/SQL/SQL.schema.jsx:135]: until @EndTime reaches the current date.

File: src/client/South/Form/__snapshots__/SouthForm.spec.jsx.snap [hunk 4]
```diff
@@ -7332,15 +7332,42 @@ exports[`SouthForm check SouthForm with dataSource: SQL 1`] = `
...
+                    To avoid a heavy query on a server, the query can be split into several sub-queries. To do so, you can use the variables @StartTime and @EndTime.
                     <br />
-                    Note for Oracle: Connection timeout can be specified in the 'sqlnet.ora' file (E.g. in /opt/oracle/instantclient_19_5/network/admin/sqlnet.ora) like this: 'SQLNET.OUTBOUND_CONNECT_TIMEOUT=500 ms'
+                    If you request data with a column named timestamp, the WHERE clause can look like this : WHERE timestamp &gt; @StartTime and timestamp &lt; @EndTime.
+                  </p>
+                  <p>
+                    @StartTime is set to the last retrieved value (retrieved from the 'Time column' field), and @EndTime is set to @StartTime + maxReadInterval (if @StartTime + maxReadInterval &lt; current date). @StartTime is then incremented at each sub-query until @EndTime reaches the current date. Each sub-query stores its results in its own csv file. See the File Settings help section bellow for more information.
```
Comment (jfhenon) [inline src/client/South/Form/__snapshots__/SouthForm.spec.jsx.snap:7367]: below

File: src/client/South/Form/__snapshots__/SouthForm.spec.jsx.snap [hunk 5]
```diff
@@ -7332,15 +7332,42 @@ exports[`SouthForm check SouthForm with dataSource: SQL 1`] = `
                   class="col"
                 >
                   <p>
-                    The query may have a specific format and contain a WHERE clause with the date constraint of the time column using @StartTime and @EndTime. If @StartTime and @EndTime are used, the query will be split according to the maxReadInterval parameter. For example, if @EndTime and @StartTime is a two hours interval, and maxReadInterval is 3600s, the query will be split in two parts. Two files will be created.
+                    The query must be adapted to the selected driver (MSSQL, Oracle...). The following variable can be used in the query:
```
Comment (jfhenon) [inline src/client/South/Form/__snapshots__/SouthForm.spec.jsx.snap:7335]: variables

**Review by jfhenon — APPROVED** [review]
2 small typos need fix

**Pull request merged**